serve/tests/unittest/test_sampler.py

```diff
@@ -8,21 +8,21 @@
 vocab_size = 32000
 
 
-def get_sampling_metadata(sampling_params, past_output_tokens=None):
+def get_sampling_state(sampling_params, past_output_tokens=None):
     batch_size = len(sampling_params)
     if past_output_tokens is None:
         past_output_tokens = [[] for _ in range(batch_size)]
     _copy_stream: torch.cuda.Stream = torch.cuda.Stream()
     with torch.cuda.stream(_copy_stream):
-        sampling_metadata = SamplingState.from_sampling_params(
+        sampling_state = SamplingState.from_sampling_params(
             sampling_params,
             list_past_output_tokens=past_output_tokens,
             dtype=dtype,
             dev=dev,
             vocab_size=vocab_size,
         )
     torch.cuda.current_stream().wait_stream(_copy_stream)
-    return sampling_metadata
+    return sampling_state
 
 
 def _test_temperature(temp=0, batch_size=1):
```

Comment:
@sunggg why do we have _ symbol before test? I added such symbol to functions which cannot be run using pytests, but functions from test_sampler.py does not require any compiled model and can be called under pytest

If there is no special reason, I propose to remove underscore symbol

Reply:
We can ask Iliya to do it in his PR (#200) due to he extends these tests now.